src/tribler/core/exceptions.py

```diff
@@ -8,9 +8,6 @@
 class TriblerException(Exception):
     """Super class for all Tribler-specific Exceptions the Tribler Core throws."""
 
-    def __str__(self):
-        return str(self.__class__) + ': ' + Exception.__str__(self)
```

Comment:
There is no point in handling the conversion to `str` for some exceptions differently from all other exceptions